projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -314,6 +314,14 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
         this.scrollTo(this.state.startIndex - 1);
     }
 
+    public scrollNextPage() {
+        this.vh.instance.elementRef.nativeElement.scrollTop += parseInt(this.igxForContainerSize, 10);
```

Comment:
if this is public, it should work for horizontal virtualization as well, changing scrollLeft